### ISSUE TITLE
[Bug]: $createUserClosure must not be accessed before initialization

### ISSUE THREAD

**Issue by Cybrarist** [chrisreedio/socialment#62]
### What happened?

the application goes to google and redirects back, then the following errors shows
```
Typed property ChrisReedIO\Socialment\SocialmentPlugin::$createUserClosure must not be accessed before initialization
```

### How to reproduce the bug

require the plugin.
`composer require chrisreedio/socialment`

install the plugin
` php artisan socialment:install`

require the provider
`composer require socialiteproviders/google`

add the event listener, and the plugin to the admin panel
```php
 protected $listen = [     
        Registered::class => [
            SendEmailVerificationNotification::class,
        ],
        \SocialiteProviders\Manager\SocialiteWasCalled::class => [
            // ... other providers
            \SocialiteProviders\Google\GoogleExtendSocialite::class."@handle",
        ],];
```
update tailwind configs
` "./vendor/chrisreedio/socialment/resources/**/*.blade.php",`

add the pluging to the panel

```php
\ChrisReedIO\Socialment\SocialmentPlugin::make()->registerProvider(
     'google' , 'fab-google','Google' ),
```

trying to login

### Package Version

v3.9.0

### PHP Version

8.2

### Laravel Version

10.45.1  and 10.47.0

### Which operating systems does with happen with?

macOS, Linux

### Notes

suggested fix in file
`vendor/chrisreedio/socialment/src/SocialmentPlugin.php:33`

update it to the following:
```php
public ?Closure $createUserClosure=null;
```

**Comment by altomar** [comment]
The fix also worked for me 🙏

**Comment by looksystems** [comment]
Same problem here - the fix works for me too.

**Comment by chrisreedio** [comment]
Hi @Cybrarist . Thanks for your bug submission. 

I had been on vacation for the past week or so. As expected, many things piled up while I was gone.

I will look to get this fixed ASAP. 

Sorry this was missed!

**Comment by chrisreedio** [comment]
This has been fixed in [v3.9.1](https://github.com/chrisreedio/socialment/releases/tag/v3.9.1)

🚀
